tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java

```diff
@@ -161,11 +171,35 @@ public void beginTask( String taskName, int totalWork )
                         {
                             final List<Problem> problems = m.findProblems( location.toFile(), override );
 
-                            problems.addAll( problems );
+                            allProblems.addAll( problems );
                         }
                     }
 
-                    m.reportProblems( allProblems, Migration.DETAIL_LONG, "ide" );
+                    if( allProblems.size() > 0 )
+                    {
+                        CodeMigrationProblems cp = new CodeMigrationProblems();
```

Comment:
Don't we need to preserve the previous migration problems?  Lets say I have 2 projects and I run "find problems" on one of them and it saves as MigrationProblems and then I run this again and select the 2nd project, wont it wipe out the problems from the first project?